ember-app/app/components/settings/hb-label-chooser.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+
+var HbLabelChooserComponent = Ember.Component.extend({
+  classNames: ["hb-label-chooser"],
+  isVisible: Ember.computed.gt("labels.length", 0),
+  applyColor: function(){
+    if(this.get("labels.length")){
+      $("div.hb-label-container").each(function(i,el){
+        var color = Ember.$(el).data("color");
+        Ember.$(el).css("background", `#${color}`);
+        Ember.$(el).css("border-color", `#${color}`);
+      });
```

Comment:
doesn't feel like `ember` to me, we should prefer property binding to the style attribute